src/models/connectionCredentials.ts

```diff
@@ -28,6 +28,7 @@ export class ConnectionCredentials implements IConnectionInfo {
 	public trustServerCertificate: boolean | undefined;
 	public hostNameInCertificate: string | undefined;
 	public persistSecurityInfo: boolean | undefined;
+	public columnEncryptionSetting: string | undefined;
```

Comment:
any need to restrict the possible values with a string literal type?

Reply:
Given that other properties with a limited value-set (e.g. applicationIntent, authenticationType) are _not_, it seems like that isn't the assumption here.  Maybe something to look into changing, but might be more than just this PR.